jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/RuntimeDataServiceImpl.java

```diff
@@ -811,6 +811,25 @@ public UserTaskInstanceDesc getTaskById(Long taskId) {
         return null;
 	}
 
+    @Override
+    public UserTaskInstanceDesc getTaskByIdWithSLA(Long taskId) {
+        Map<String, Object> params = new HashMap<String, Object>();
+        params.put("taskId", taskId);
+        params.put("maxResults", 1);
+        List<UserTaskInstanceDesc> tasks = commandService.execute(
+                new QueryNameCommand<List<UserTaskInstanceDesc>>("getTaskInstanceById", params));
```

Comment:
This looks like an exact copy of `getTaskById`. Can we perhaps reuse that method in this new one?

Reply:
@MarianMacik yes, I updated the PR reusing getTaskById. Thanks!